test/runUnitTests.js

```diff
@@ -16,4 +16,10 @@
 
 import {unitTestRunner} from './unit/unitTestRunner.js';
 
-unitTestRunner.parseOptionsAndRun();
\ No newline at end of file
+unitTestRunner.run().catch((ex) => {
+  if (typeof ex === 'number') {
```

Comment:
How do you get here? You are resolving the promise with the number of failures and not rejecting it so catch will not be invoked.